packages/containers/src/DeleteResource/index.js

```diff
@@ -0,0 +1,14 @@
+import DeleteResource from './DeleteResource.connect';
+import deleteResourceConstants from './deleteResource.constants';
+import deleteResourceSagas from './deleteResource.sagas';
+import actions, { validate, cancel } from './deleteResource.actions';
```

Comment:
why do you import alone validate and cancel and not open. import actions only and use actions.open, actions.validate, actions.cancel.